packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -986,13 +986,16 @@ function commitContainer(finishedWork: Fiber) {
     }
     case HostRoot:
     case HostPortal: {
+      const portalContainer = finishedWork.stateNode;
       const portalOrRoot: {
         containerInfo: Container,
         pendingChildren: ChildSet,
         ...
-      } = finishedWork.stateNode;
+      } = portalContainer;
       const {containerInfo, pendingChildren} = portalOrRoot;
       replaceContainerChildren(containerInfo, pendingChildren);
+      // Update the fiber on the portal container
+      portalContainer.current = finishedWork;
```

Comment:
This seemed like the best place to update the `current` pointer to the most recent fiber on the portal container.

Reply:
This is only for the "persistent" mode (i.e. Fabric). There's no commit phase for Portals.